cpp/ql/test/library-tests/ir/ir/ir.cpp

```diff
@@ -2686,4 +2686,3102 @@ void test(bool b)
   twice_call_use(b ? "" : "");
 }
 
+#define CONCAT(a, b) CONCAT_INNER(a, b)
+#define CONCAT_INNER(a, b) a ## b
+
+#define READ do { String CONCAT(x, __COUNTER__); } while(0)
+
+#define READ2 READ; READ
+#define READ4 READ2; READ2
+#define READ8 READ4; READ4
+#define READ16 READ8; READ8
+#define READ32 READ16; READ16
+#define READ64 READ32; READ32
+#define READ128 READ64; READ64
+#define READ256 READ128; READ128
+#define READ512 READ256; READ256
+#define READ1024 READ512; READ512
+#define READ1025 READ1024; READ
+
+void many_defs_per_use() {
+  // This code is the result of preprocessing `READ1025`. The extractor
+  // currently outputs references to unknown labels when using the unexpanded
+  // macro, however. So for now we expand it manually.
```

Comment:
Should we just remove the macros and move this test into its own file?

Reply:
We can do that, sure 👍

Reply:
Fixed in 099c282277a1c180ff3ce8ef555b826b848ab617